server/controllers/finance/reports/break_even/report_project.handlebars

```diff
@@ -0,0 +1,313 @@
+                  </i>                  
+                </td>
+                <td class="text-right" style="width: 20%; font-size: 15px">
+                  <strong>
+                    {{debcred data.resultsTurnOverP ../metadata.enterprise.currency_id}}
+                  </strong>                  
+                </td>
+              </tr>
+              <tr>
+                <td class="text-right" style="width: 30%; font-size: 15px">
+                  <i>
+                    {{translate "TABLE.COLUMNS.BREAK_EVEN.MARGINS_VARIABLE_EXPENSES" }}
+                  </i>                  
+                </td>
+                <td class="text-right" style="width: 20%; font-size: 15px">
+                  <strong>
+                    {{debcred data.ratioMarginVariableLoadsP ../metadata.enterprise.currency_id}}
+                  </strong>                  
+                </td>
+              </tr>
+              <tr>
+                <td class="text-right" style="width: 30%; font-size: 15px">
+                  <strong><i>
+                    {{translate "TABLE.COLUMNS.BREAK_EVEN.BREAK_EVEN" }}
+                  </i></strong>                  
+                </td>
+                <td class="text-right" style="width: 20%; font-size: 15px">
+                  <strong>
+                    {{debcred data.breakEvenValueP ../metadata.enterprise.currency_id}}
+                  </strong>
+                </td>
+              </tr>
+              <tr>
+                <td class="text-right" style="width: 30%; font-size: 15px">
+                  <strong><i>
+                    {{translate "TABLE.COLUMNS.BREAK_EVEN.BREAK_EVEN_POINT" }} 
+                  </i></strong>                  
+                </td>
+                <td class="text-right" style="width: 20%; font-size: 15px">
+                  <em>
+                    {{ data.breakEvenPointP }} {{translate "TABLE.COLUMNS.DAYS"}}
+                  </em>
+                </td>
+              </tr>
+              <tr>
+                <td class="text-right" style="width: 30%; font-size: 15px">
+                  <strong><i>
+                    {{translate "TABLE.COLUMNS.BREAK_EVEN.BREAK_EVEN_POINT" }}
+                  </i></strong>                  
+                </td>
+                <td class="text-right" style="width: 20%; font-size: 15px">
+                  <strong>
+                    {{ data.breakEvenPointCaseP }} {{translate "TABLE.COLUMNS.CASES"}}
+                  </strong>
+                </td>
+              </tr>
+              {{/if}}
+            </tbody>
+            <tfoot>
+              {{#if data.cantCalculateBreakEven}}
+              <tr style="width: 40%; font-size: 15px; text-align: center;" colspan="3" scope="row">
+                <td>
+                  <strong>
+                    {{translate "TABLE.COLUMNS.BREAK_EVEN.DIFFERENTIAL_INCOME_STATEMENT" }} <SUB> (1) </SUB>
+                  </strong>                     
```

Comment:
Could you add in the bottom of the document the meaning or definition of `(1)` and `(2)`